cdip_connector/core/connector_base.py

```diff
@@ -38,6 +61,8 @@ def execute(self) -> None:
     async def main(self) -> None:
         try:
             integrations = await self.portal.get_authorized_integrations()
+            integrations = filter_items_for_task(integrations)
+
```

Comment:
There is a known issue in this main method of the connector (just in case you want to take the opportunity to fix that too). If one of the integrations in the batch raises an exception then that breaks the for loop and other batches are not processed (other integrations in the same batch continue to be processed so this affects integrations where we have many like GFW)
You can see a fix we did in GFW here:
https://github.com/PADAS/cdip-integrations/blob/2730b830a92eb24042442859244f572bb7cada74/gfw/gfw_connector.py#L112

Reply:
Thanks @marianobrc I failed to see this message last month. I'll make this fix now and send an update.